### PR TITLE
Add SSOMetadataURL example

### PR DESCRIPTION
**Pull request by nwneisen**
#71 

Adds an example for the SSOMetaDataURL to the Okta setup.

Created tabs and added more details for manual configuration

Fixed some invalid redirect urls

Replaces #79

### CODE REVIEW

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 1]
```diff
@@ -28,7 +28,100 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+</md:EntityDescriptor>
+```
+
+{{< /tab >}}
+
+{{< tab >}}
+
+Manually configuring the SAML service in MKE allows more control over the configuration.
+
+```yaml
```
Comment (KoryKessel-Mirantis) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:80]:
```suggestion
An example of the `saml` section of the MKE 4 configuration file under `authentication` follows:

```yaml
```

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 2]
```diff
@@ -28,7 +28,100 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+4. The `ssoURL` is displayed in the `Identity Provider Single Sign-On URL` section.
+5. The certificate is displayed in the `X.509 Certificate` section.
+6. The `redirectURI` will be the external address of your cluster with `/dex/callback` appended to the end.
+7. The `usernameAttr` and `emailAttr` are the attributes that will be used to map the user's name and email to the MKE user. These can be left the same as the values in the example since that is what Okta was configured to use above.
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+Run the `mkectl apply` command with your MKE configuration file.
```
Comment (KoryKessel-Mirantis) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:124]:
```suggestion
Once the `saml` section of the MKE 4 configuration file is set, run it with the `mkectl apply` command.
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 3]
```diff
@@ -28,7 +28,95 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+```yaml
+authentication:
+  saml:
+    enabled: true
+    ssoMetadataURL: https://dev-14305804.okta.com/app/exkdtjvsinbvwD5ms5d0/sso/saml/metadata
+```
+
+To obtain the metadata URL in Okta, navigate to **Applications -> Applications -> Your application** and click the **Sign On** tab.
+
+Example of information provided when you access the `ssoMetadataURL` URL:
```
Comment (nwneisen) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:48]:
```suggestion
Example ssoMetadataURL information:
```

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 4]
```diff
@@ -28,7 +28,100 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+</ds:X509Data>
+</ds:KeyInfo>
+</md:KeyDescriptor>
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</md:NameIDFormat>
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress</md:NameIDFormat>
+<md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://dev-64105006.okta.com/app/dev-63105106_mke_2/exk75pi5do2MzU1t95r7/sso/saml"/>
+<md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://dev-63105106.okta.com/app/dev-63105106_mke_2/exk75pi5do2MzU1t95r7/sso/saml"/>
+</md:IDPSSODescriptor>
+</md:EntityDescriptor>
+```
```
[the remark this answers is not in the document]
Comment (nwneisen) [reply content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md]: Carry over the discussion from the previous PR

This is useful for a customer when they are setting up SAML and they wonder what they should be seeing. If the goal of the docs is to "help customers setup an MKE4 cluster", then I suggest we leave this and look for more areas that we can clarify what should be seen for customers.

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 5]
```diff
@@ -28,7 +28,95 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+
+{{< /tab >}}
+
+{{< tab >}}
+
+Set up the SAML service manually in MKE to gain more control over the configuration.
+An example of the `saml` section of the MKE 4 configuration file under `authentication` follows:
```
Comment (KoryKessel-Mirantis) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:73]:
```suggestion

An example of the `saml` section of the MKE 4 configuration file under `authentication` follows:
```

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 6]
```diff
@@ -28,7 +28,95 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+```yaml
+authentication:
+  saml:
+    enabled: true
+    ssoMetadataURL: https://dev-14305804.okta.com/app/exkdtjvsinbvwD5ms5d0/sso/saml/metadata
+```
+
+To obtain the metadata URL in Okta, navigate to **Applications -> Applications -> Your application** and click the **Sign On** tab.
+
+Example ssoMetadataURL information:
```
Comment (KoryKessel-Mirantis) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:48]:
```suggestion
Example `ssoMetadataURL` information:
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

**Review by KoryKessel-Mirantis — APPROVED** [review]
Passing to @MagdaDziadosz for second TW review.

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 7]
```diff
@@ -28,7 +28,96 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
+|  `ssoURL`      | Obtain the `ssoURL` and certificate information from Okta:<br>  1. Navigate to **Applications -> Applications -> Your application**.<br>  2. Click the **Sign On** tab.<br>  3. Click the **View SAML setup instructions` link**. The `ssoURL` displays in the `Identity Provider Single Sign-On URL` section. |
+| `redirectURI`  | The external address of your cluster, with `/dex/callback` appended.                                                                                                                                                                                                                                           |
+| `usernameAttr` | Attribute used to map the username to the MKE user.                                                                                                                                                                                                                                                            |
```
Comment (MagdaDziadosz) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:112]:
```suggestion
| `usernameAttr` | Attribute that maps the username to the MKE user.                                                                                                                                                                                                                                                            |
```
Changed to match the attribute description below.

File: content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md [hunk 8]
```diff
@@ -28,7 +28,96 @@ To configure an Okta application to serve as your [SAML authentication](../../..
...
     a. Navigate to the SAML **Signing Certificates** section.
     b. Click **Actions** for the active certificate and initiate the download.
-11. Run the `mkectl apply` command with your MKE configuration file.
+
+## Configure MKE
+
+{{< tabs items="SSO metadata URL,manual" >}}
+
+{{< tab >}}
+Okta provides a metadata URL with which you can configure SAML for MKE 4. MKE is able to obtain information for all SAML configurations in your MKE cluster through that URL, which you configure to the `ssoMetadataURL` parameter in the `authentication` section of the MKE configuration file.
```
Comment (MagdaDziadosz) [inline content/docs/tutorials/authentication-provider-setup/setting-up-okta-as-a-saml-provider.md:37]:
```suggestion
Okta provides a metadata URL with which you can configure SAML for MKE 4.
MKE is able to obtain information for all SAML configurations in your MKE cluster through that URL,
which you configure to the `ssoMetadataURL` parameter in the `authentication` section of the MKE
configuration file.
```
Broken between few lines to meet our style guidelines.

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**